### DISCUSSION TITLE
Loading 1 new model take a very longtime

### DISCUSSION THREAD

**Discussion by giang17tn1** [lllyasviel/Fooocus#2280] [Q&A] [answered]
Dear team,

I just move swap space to SSD with 50GB of space available. But it's take total 720s for each image. I am using 16 GB RAM with GTX 1660 6GB VRAM. So how could I fix this?

![image](https://github.com/lllyasviel/Fooocus/assets/160092911/4ff05ba6-c6ce-40e7-ae58-08f5a60a9f4d)

**Comment by mashb1t** [comment, marked as answer]
see https://github.com/lllyasviel/Fooocus/discussions/2274
